### PR TITLE
[Term Entry] Lua Table Function: unpack()

### PR DESCRIPTION
**Pull request by phuongtrieu97coder**
### Description

- Added the unpack() term for Lua
- Closes issue #3171  

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/tables/terms/unpack/unpack.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`unpack()`** function takes a table and returns the elements of that table as separate values.
+
+## Syntax
+
+```pseudo
+result1, result2, ..., resultn = unpack(table)
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/unpack/unpack.md:20]:
```suggestion
result1, result2, ..., resultn = table.unpack(someTable)
```

File: content/lua/concepts/tables/terms/unpack/unpack.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+The example will result in the following output:
+
+```shell
+Type1:  orange
+Type2:  apple
+```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/lua/concepts/tables/terms/unpack/unpack.md:44]: This codebyte is demonstrating the same thing as the previous example. Update the code to show some new functionality or delete the codebyte

**Review by caupolicandiaz — APPROVED** [review]
@phuongtrieu97coder I updated the spacing in the examples and change the second code block to a standard example. I don't believe our compiler currently supports Lua. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@phuongtrieu97coder Looks good.

**Pull request merged**